cfme/containers/container.py

```diff
@@ -41,6 +41,16 @@ def get_detail(self, *ident):
         self.load_details(refresh=True)
         return details_page.infoblock.text(*ident)
 
+    @staticmethod
+    def get_container_names():
+        navigate_to(Container, 'All', use_resetter=False)
```

Comment:
If not, we can always make the "All" destination go to Table View and not Grid View